pkgs/servers/pulseaudio/default.nix

```diff
@@ -88,7 +95,7 @@ stdenv.mkDerivation rec {
   );
 
   mesonFlags = [
-    "-Dalsa=${if !libOnly then "enabled" else "disabled"}"
+    "-Dalsa=${if !libOnly && stdenv.isLinux then "enabled" else "disabled"}"
```

Comment:
I think we should move that into an assert that libOnly does not work on darwin

Reply:
libpulseaudio works well on darwin.

Reply:
I think it's more appropriate to move the logic to, perhaps, alsaSupport?

Reply:
Maybe, to make it less confusing

Reply:
Done.